native/src/components/DatesPageDetail.tsx

```diff
@@ -1,21 +1,35 @@
-import React, { ReactElement } from 'react'
+import React, { ReactElement, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { View } from 'react-native'
 import styled from 'styled-components/native'
 
-import { MAX_DATE_RECURRENCES, MAX_DATE_RECURRENCES_COLLAPSED } from 'shared'
+import { MAX_DATE_RECURRENCES } from 'shared'
 import { DateModel } from 'shared/api'
 
-import Collapsible from './Collapsible'
+import { CalendarTodayIcon, ClockIcon, ExpandIcon } from '../assets'
 import PageDetail from './PageDetail'
+import Icon from './base/Icon'
+import Pressable from './base/Pressable'
 import Text from './base/Text'
 
-const Identifier = styled(Text)`
-  font-weight: bold;
-  color: ${props => props.theme.colors.textColor};
+const SingleDateContainer = styled.View`
+  margin-bottom: 8px;
 `
-const StyledText = styled(Text)`
-  color: ${props => props.theme.colors.textColor};
+
+const StyledPressable = styled(Pressable)`
```

Comment:
This button is an interim solution until the new design system is implemented, and has been approved by Toni

<img width="117" height="34" alt="Image" src="https://github.com/user-attachments/assets/29899236-7af9-4179-8c27-396a9c76c300" />